docs/whatsnew/3.0.rst

```diff
@@ -158,6 +176,23 @@ or not to use memory mapping when reading the table, and ``fits.open`` supports
 a new keyword argument ``character_as_bytes`` to return character columns as
 Numpy byte arrays (Numpy type 'S'), which uses less memory.
 
+.. _whatsnew-3.0-ascii-chunks:
+
+Reading large ASCII files in chunks
+===================================
+
+Astropy now supports reading very large ASCII tables in smaller chunks which are
+limited in size.  There are two possible ways to do this:
```

Comment:
Since the rushed user might ask "why do I care?", perhaps insert a sentence after the first one along the lines of "This allows tables to be loaded that approach the maximum memory available."?